extra_modules_python/DiscontinuityLayer.py

```diff
@@ -0,0 +1,124 @@
+import radiopropa
```

Comment:
This file can be removed.

Reply:
clean up is done in the next pull request ;)